config/dev.cfg

```diff
@@ -16,7 +16,8 @@ credential.password=testPwd
 
 [path]
 path.ebooks=ebooks
-path.extras=ebooks/extras
+path.extras=extras
+path.separate=true
```

Comment:
In order to avoid any regression, I suggest to
```
# leave the old path: unfortunately I have written no tests... my fault
path.extras=ebooks/extras

# add something like the following, default must be FALSE
path.separate=false
path.separate.ebooks=MY/EBOOKS/FOLDER
path.separate.extras=MY/EXTRAS/FOLDER
```